src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -561,47 +603,28 @@ module.exports = function<T, P, I, TI, C>(config : HostConfig<T, P, I, TI, C>) {
       try {
         performTaskWorkUnsafe();
       } catch (error) {
-        isPerformingTaskWork = false;
```

Comment:
I moved this one into a try/finally in performTaskWorkUnsafe so that it's only set there.